CONTRIBUTING.md

```diff
@@ -32,6 +32,8 @@ maximize the chances of your PR being merged.
   Please see [envoy/support/README.md](https://github.com/envoyproxy/envoy/tree/master/support) for
   more information on these hooks.
 
+  Note: the git hooks depend upon many of the dependencies used to build envoy via Bazel. Follow the [Quick start Bazel build for developers](https://github.com/envoyproxy/envoy/blob/master/bazel/README.md#quick-start-bazel-build-for-developers) guide to install these.
```

Comment:
Would you mind breaking this into multiple lines like the rest of the file so as not to exceed 110 characters if possible?

Reply:
I think you may have accidentally dropped the commit when rebasing

Reply:
yah, I retracted the commit simply breaking it to two lines. but if it's ok for URLs to overflow, then let's go with that